run/src/main/kotlin/com/cosmotech/run/workflow/argo/RunArgoWorkflowService.kt

```diff
@@ -232,17 +224,27 @@ internal class RunArgoWorkflowService(
         workflowList =
             newServiceApiInstance<ArchivedWorkflowServiceApi>(this.apiClient)
                 .archivedWorkflowServiceListArchivedWorkflows(
```

Comment:
Why are we still looking for archived workflow separately if they are now listed as regular workflows?